apps/aecore/lib/aecore/channel/channel_state_peer.ex

```diff
@@ -1060,7 +1125,7 @@ defmodule Aecore.Channel.ChannelStatePeer do
           poi: dispute_poi_for_latest_state(peer_state),
           offchain_tx: ChannelTransaction.dispute_payload(most_recent_tx)
         },
-        our_pubkey(peer_state),
+        pubkey,
```

Comment:
If the role is :initiator or :responder we should check that pubkey == our_pubkey(peer_state)